Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      description: The maximum number of results to return. Default is 50.
+      isArray: false
+      name: limit
+      required: false
+    - default: false
+      isArray: false
+      name: page
+      description: Page number you would like to view. Each page contains page_size values. Must be used along with page_size.
+    - default: false
+      isArray: false
+      name: page_size
+      description: Number of results per page to display.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Lists all configuration jobs.
+    execution: false
+    name: prisma-sase-config-job-list
+    outputs:
+    - contextPath: PrismaSase.ConfigJob.description
+      description: Configuration job description.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.end_ts
+      description: Configuration job end timestamp.
+      type: Date
+    - contextPath: PrismaSase.ConfigJob.id
+      description: Configuration job ID.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.job_result
+      description: Configuration job result.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.job_status
+      description: Configuration job status.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.job_type
+      description: Configuration job type.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.parent_id
+      description: Configuration job parent ID.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.percent
+      description: Configuration job percent.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.result_str
+      description: Configuration job result string.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.start_ts
+      description: Configuration job start timestamp.
+      type: Date
+    - contextPath: PrismaSase.ConfigJob.status_str
+      description: Configuration job status string.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.summary
+      description: Configuration job summary.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.type_str
+      description: Configuration job type string.
+      type: String
+    - contextPath: PrismaSase.ConfigJob.uname
+      description: Configuration job uname.
+      type: String
+  - arguments:
+    - default: false
+      description: Rule ID of the rule to be deleted.
+      isArray: false
+      name: rule_id
+      required: true
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Delete a specific security rule.
+    execution: false
+    name: prisma-sase-security-rule-delete
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or the type of folder?